test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,22 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  test("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(3)).toBe("Fizz");
+  });
 
-  test("returns Buzz when passed a multiple of 5", () => {});
+  it("returns Buzz when passed a multiple of 5", () => {
```

Comment:
This should be updated to `test` so it matches the others. You could use either of these words tbh, its up to you, but whichever you choose I would make sure to use the same work throughout